File: web2projekat/src/models/ArticleDto.js
export default class ArticleDto {
    constructor(data) {
        this.Id = data.id;
        this.Name = data.name;
        this.Price = data.price; 
        this.Quanity = data.quanity; 
        this.Description = data.description; 
        this.Image = data.image;
        this.UserId = data.userId;
    }
}

File: WebServer/WebServer/Dto/ArticleDto.cs
using System.ComponentModel.DataAnnotations;

namespace WebServer.Dto
{
    public class ArticleDto
    {
        [Required]
        public long Id { get; set; }
        [Required]
        public string Name { get; set; }
        [Required]
        public double Price { get; set; }
        [Required]
        public int Quanity { get; set; }
        [Required]
        public string Description { get; set; }
        public string Image { get; set; }
        [Required]
        public long UserId { get; set; }
    }
}
